test/verify/check-shell-keys

```diff
@@ -304,18 +304,17 @@ session    optional     pam_ssh_add.so
         b.wait_visible("#credential-keys")
         b.wait_not_present("#ssh-file-add")
         b.click("#ssh-file-add-custom")
-        b.set_file_autocomplete_val("#ssh-file-add-key", "/bad")
+        b.set_file_autocomplete_val("#ssh-file-add-key", "/etc/")
```

Comment:
Why the path change? Unlike in the previous comment, this makes an actual difference -- /etc/ exists, but /bad doesn't.

Reply:
Yes, exactly. The FileAutocomplete widget does no longer allow selecting non-existing files (or directories), unless isCreatable is true.

Reply:
Thanks!